packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -165,21 +170,29 @@ export default class MeetingRequest extends StatelessWebexPlugin {
 
   /**
    * Make a network request to leave a meeting
-   * @param {Object} options
-   * @param {String} options.deviceUrl
-   * @param {String} options.resourceId,
-   * @param {String} options.correlationId
+   * @param {Url} locusUrl
```

Comment:
(we discussed this on call but leaving here for posterity) Let's add the `options.` back to show they are destructured params.